apps/omg_eth/lib/omg_eth/wait_for.ex

```diff
@@ -69,4 +69,18 @@ defmodule OMG.Eth.WaitFor do
       :error, {:badmatch, _} = _error -> repeat_until_ok(f)
     end
   end
+
+  def repeat_until_ok(f, acc) do
```

Comment:
is there a limit the amount of times this would get called until it should time out?

Reply:
Is this function used anywhere else then in `watcher_synchronize`? why not included it there and limit the fragmentation?